modules/twinkleblock.js

```diff
@@ -1040,6 +1040,14 @@ Twinkle.block.blockPresetsInfo = {
 		reasonParam: true,
 		reason: 'Email [[WP:Harassment|harassment]]',
 		summary: 'You have been [[WP:PB|blocked]] from emailing other editors for [[WP:Harassment|harassment]]'
+	},
+	'uw-aepblock': {
+		autoblock: true,
+		nocreate: false,
+		pageParam: false,
+		reason: '[[WP:Arbitration enforcement|Arbitration enforcement]]',
+		reasonParam: true,
+		summary: 'You have been [[WP:PB|blocked]] from editing for violating an [[WP:Arbitration|arbitration decision]] with your edits'
```

Comment:
Change to "...arbitration sanction." The last bit is hanging.

Reply:
@QEDK You mean:

```suggestion
		summary: 'You have been [[WP:PB|blocked]] from editing for violating an [[WP:Arbitration|arbitration sanction]]'
```

I'm fine with removal of "with your edits" — it was copied from the summary for uw-aeblock and agree it's weird — but I think it should stay as "decision": it's the language used by the actual template, and the block itself is the "sanction".  Yeah, technically, you can be sanctioned for violating a sanction, but it's a little jargony and unnatural English, and not 100% of cases.

Reply:
> @QEDK You mean:
> 
> I'm fine with removal of "with your edits" — it was copied from the summary for uw-aeblock and agree it's weird — but I think it should stay as "decision": it's the language used by the actual template, and the block itself is the "sanction". Yeah, technically, you can be sanctioned for violating a sanction, but it's a little jargony and unnatural English, and not 100% of cases.

That should do as well.